packages/terra-core-docs/src/terra-dev-site/doc/arrange/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,146 @@
+# Accessibility Guide for Terra Arrange
+
+
+## Why the accessibility of Arrange is important ?
+
+> Arrange is a component which is used for performing the layout alignment in the position of top/bottom/right/left. Though the CSS alignment does not affect the accessibility, the reading order of the content need to be maintained same as the visual order. If the content is aligned with any position property to make it look in different place which will create a complexity for the screen reader user to get the correct reading order.   
+>
+> Making the DOM order same as visual order will help the assistive technology users to get the correct reading order. 
+>
+> Arrange may also have the interactive elements which needs to be keyboard focusable and acted upon keyboard Enter/Space key. 
+>
+>&nbsp;&nbsp;
+- **Blind screen reader users** do not necessarily interact with the page in a linear fashion and can use our component out of context. Hence, we need to provide descriptive label for each interactive element. 
```

Comment:
Not all screen reader users are blind :)

```suggestion
- **Screen reader users** do not necessarily interact with the page in a linear fashion and can use Arrange out of context. Hence, a descriptive label is necessary for each interactive element. 
```

Reply:
We can do this minor change

Reply:
Updated.